sdk/python/kubeflow/training/api/training_client_test.py

```diff
@@ -203,6 +205,119 @@ def __init__(self, kind) -> None:
         self.kind = kind
 
 
+def mock_get_job_pods(*args, **kwargs):
+    """Mock get_job_pods to return controlled pod objects"""
+    namespace = kwargs.get("namespace")
+    if namespace == f"pod {TIMEOUT}":
```

Comment:
Why this namespace has `pod` in the name ?

Reply:
I would like to differentiate the timeout error from different sources, one is from `get_job_pods`, and the other one is from `get_job`.